### PR TITLE
RemovedInDjango20Warning: `Options.virtual_fields` is deprecated

### PR DESCRIPTION
**Pull request by luiscastillocr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ MANIFEST
 __pycache__
 /coverage/
 /.coverage
+.idea/
```
Comment (auvipy) [inline .gitignore]: this is not related

Comment (luiscastillocr) [reply .gitignore]: Wouldn't you like to have the JetBrains project directory ignored>

**Comment by luiscastillocr** [comment]
Without Mock in the project I don't see a way to test it, any suggestion is welcome
